src/components/course/EnrollModal.jsx

```diff
@@ -93,7 +92,7 @@ const EnrollModal = ({
           href={enrollmentUrl}
           onClick={handleEnroll}
         >
-          {isLoading && <FontAwesomeIcon icon={faSpinner} alt="loading" className="fa-spin mr-2" />}
+          {isLoading && <Cached className="mr-2" svgAttrs={{ 'aria-label': 'loading' }} />}
```

Comment:
[curious] Just curious about your preference for using the icon as a React component with `svgAttrs` here versus using the `Icon` component, passing `Cached` as its `src` prop.

```jsx
<Icon src={Cached} screenReaderText="loading" />
```

I believe `aria-label` is primarily intended for forms or interactive elements, whereas the `screenReaderText` above gets renders in an element with `.sr-only` (i.e., visually hidden text, but will be read aloud to screen readers).